virttest/guest_agent.py

```diff
@@ -730,3 +725,32 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    @error_context.context_aware
+    def file_operate(self, operate, **kwargs):
+        """
+        file operation function,
+        :param operate: open,write,read,flush,seek and flush
+        :param kwargs: optional keyword arguments including but not limited to below
+        :keyword Args:
+                handle: file handle returned by guest-file-open
+                mode: optional open mode, "r" is the default value.
+                content: content to write
+                count: optional,maximum number of bytes to read (default is 4KB)
+                offset: offset of whence
+                whence: 0,file beginning;
+                        1,file current position;
+                        2,file end
+        :return: The command's output
+        """
+        cmd = "guest-file-%s" % operate
+        if operate == "read" and kwargs["count"] is None:
+            kwargs.pop("count")
+        if operate == "write":
+            con = kwargs["content"]
+            kwargs.pop("content")
```

Comment:
@xiagao Would you help explain why pop it? Cause I remember like you have `content` together with `buf-b64` in the original version.

Reply:
The  arg in original version is _{"handle": handle, "buf-b64": base64.b64encode(content.encode()).decode()}_ ,so need  change _content:xxxx_  to _"buf-b64": base64.b64encode(con.encode()).decode()_
So first pop content and then add buf-b64

Reply:
@xiagao I see, thanks.